pages/app-layout/dashboard-content-type.page.tsx

```diff
@@ -0,0 +1,48 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { useState } from 'react';
+import AppLayout from '~components/app-layout';
+import Button from '~components/button';
+import Header from '~components/header';
+import SplitPanel from '~components/split-panel';
+import ScreenshotArea from '../utils/screenshot-area';
+import { Containers, Navigation, Tools, Breadcrumbs } from './utils/content-blocks';
+import * as toolsContent from './utils/tools-content';
+import labels from './utils/labels';
+import { splitPaneli18nStrings } from './utils/strings';
+
+export default function () {
+  const [splitPanelOpen, setSplitPanelOpen] = useState(false);
+  return (
+    <ScreenshotArea gutters={false}>
+      <AppLayout
+        contentType="dashboard"
+        ariaLabels={labels}
+        breadcrumbs={<Breadcrumbs />}
+        navigation={<Navigation />}
+        tools={<Tools>{toolsContent.long}</Tools>}
+        contentHeader={
+          <Header variant="h1" actions={<Button onClick={() => setSplitPanelOpen(true)}>Add widget</Button>}>
+            Dashboard page
+          </Header>
+        }
+        content={<Containers />}
+        splitPanelPreferences={{ position: 'side' }}
+        splitPanelOpen={splitPanelOpen}
+        splitPanel={
+          splitPanelOpen && (
```

Comment:
ideally, we could avoid this. But we have a lot of code using `if(splitPanel)` condition, which will have to be extended to `splitPanel || splitPanelCloseBehavior !== 'collapse'`.

<img width="426" alt="image" src="https://user-images.githubusercontent.com/812240/213412298-befefcc0-8057-4f3c-bc54-4bea8e784ff3.png">

Reply:
I decided not to change all of this code for now. This improvement is possible to do incrementally, if we receive that current integration is difficult

Reply:
But if customers need to write this conditional rendering logic, does the new prop bring any advantage? Since they could just do this before:

```
splitPanelOpen && (<SplitPanel {...} />)
```

In other words, with the conditional logic the `closeBehavior` property is not relevant —or am I missing something?

Reply:
> we have a lot of code using `if(splitPanel)` condition, which will have to be extended to `splitPanel || splitPanelCloseBehavior !== 'collapse'`

Does this mean that if customers use the `closeBehavior` property without conditional rendering, some things could break?

Reply:
Not break as throwing JS errors, but it will not match the expected design

Reply:
> does the new prop bring any advantage

It changes rendering of split panel itself in the open state (different close buttons on the panel itself and app layout)

Reply:
Updated